disnake/threads.py

```diff
@@ -620,6 +620,7 @@ async def edit(
         slowmode_delay: int = MISSING,
         auto_archive_duration: AnyThreadArchiveDuration = MISSING,
         pinned: bool = MISSING,
+        flags: ChannelFlags = None,
```

Comment:
```suggestion
        flags: ChannelFlags = MISSING,
```
edit methods should generally use `MISSING` for all parameter defaults (except `reason`) - this also needs some slight adjustments in the parameter handling below

Reply:
https://github.com/DisnakeDev/disnake/pull/642/commits/e62f2da4ecaa0aed7798ed81e5d803f4323074fe